x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -546,19 +551,31 @@ static void writePkcs12(String fileName, OutputStream output, String alias, Cert
                                 char[] password, Terminal terminal) throws Exception {
             final KeyStore pkcs12 = KeyStore.getInstance("PKCS12");
             pkcs12.load(null);
-            withPassword(fileName, password, terminal, p12Password -> {
-                if (isAscii(p12Password)) {
-                    pkcs12.setKeyEntry(alias, pair.key, p12Password, new Certificate[]{pair.cert});
-                    if (caCert != null) {
-                        pkcs12.setCertificateEntry("ca", caCert);
+
+        boolean olderVersion = false;
+        String command = "openssl version -a";
```

Comment:
We don't want to actually check what version of `openssl` they have installed.
They might be generating certificates to be used on a different machine that has a different version of openssl.
It's OK to always print a warning about the password exceeding the old openssl limit.